cmd/pke/app/phases/kubeadm/controlplane/controlplane.go

```diff
@@ -881,6 +883,17 @@ func (c *ControlPlane) installMaster(out io.Writer) error {
 		return err
 	}
 
+	// sed -i "/^[ \t]*- --port=0$/d" /etc/kubernetes/manifests/kube-scheduler.yaml
+	err = runner.Cmd(out, "sed", "-i", "/^[ \t]*- --port=0$/d", kubeSchedulerManifest).Run()
```

Comment:
Can you please add some explanations here? What this does and why? Also, reference an issue/PR/docs if there is a relevant one.

Reply:
After the https://github.com/kubernetes/kubernetes/pull/92720 merged the `kubectl get cs` command returned errors in the case of `scheduler` and `controller-manager` because of the insecure ports were disabled in the manifest files: 

```NAME                 STATUS      MESSAGE                                                                                     ERROR
scheduler            Unhealthy   Get http://127.0.0.1:10251/healthz: dial tcp 127.0.0.1:10251: connect: connection refused
controller-manager   Unhealthy   Get http://127.0.0.1:10252/healthz: dial tcp 127.0.0.1:10252: connect: connection refused
etcd-0               Healthy     {"health":"true"}
```
I added these `sed` lines only for testing purpose for seeing an errorless component status. I will remove these lines because the K8s cluster works properly in spite of the insecure ports are disabled in the components